src/index.html

```diff
@@ -10,9 +10,9 @@
     <title>Ylitse MentorApp - SOS-Lapsikylä</title>
     <link rel="stylesheet" type="text/css" href="styles/fonts.css" />
     <link rel="stylesheet" type="text/css" href="styles/normalize.css" />
-    <link rel="icon" href="../static/favicon/favicon.ico" sizes="32x32">
-    <link rel="icon" href="../static/favicon/icon.svg" type="image/svg+xml">
-    <link rel="apple-touch-icon" href="../static/favicon/apple-touch-icon.png">
+    <link rel="icon" href="favicon/favicon.ico" sizes="32x32">
```

Comment:
@idolgov when running build, the static assets are duplicated -> they are in static-folder, but also a copy is made one up

I opted to use the same path as the styles, so one up